django/template/backends/django.py

```diff
@@ -62,6 +63,9 @@ def origin(self):
         return self.template.origin
 
     def render(self, context=None, request=None):
+        return mark_safe(''.join(self.stream(context, request)))
```

Comment:
`join()` does return an unsafe string as well. There already are some use of `mark_safe()` with `join()` in the core done in the same way:

* https://github.com/django/django/blob/master/django/contrib/gis/maps/google/gmap.py#L215
* https://github.com/django/django/blob/master/django/contrib/admin/widgets.py#L54
* https://github.com/django/django/blob/master/django/contrib/admin/widgets.py#L182